File: sdk/python/microsandbox/hello.py
"""
Hello World module for the Microsandbox SDK.
"""


def greet(name: str) -> str:
    """
    Returns a greeting message for the given name.

    Args:
        name: The name to greet

    Returns:
        A greeting message
    """
    message = f"Hello, {name}! Welcome to Microsandbox!"
    print(message)
    return message

File: sdk/php/src/Hello.php
<?php

namespace Microsandbox;

/**
 * Hello class for the Microsandbox SDK.
 */
class Hello
{
    /**
     * Returns a greeting message for the given name.
     *
     * @param string $name The name to greet
     * @return string A greeting message
     */
    public static function greet(string $name): string
    {
        $message = "Hello, {$name}! Welcome to Microsandbox!";
        echo $message . PHP_EOL;
        return $message;
    }
}
